### PR TITLE
Add HTML generation via Webpack 

### PR DESCRIPTION
**Pull request by ogonkov**
* Generate year from JS
* Use templates for HTML generation
* Add meta viewport for correct adaptation on mobile

### CODE REVIEW

File: src/main/html/charts.ejs [hunk 1]
```diff
@@ -73,12 +70,10 @@ <h2>Download statistics</h2>
         </footer>
     </section>
     <footer id="about">
-        <p>&copy; 2012&ndash;2019 &ndash; Paul Vorbach. <a href="https://paul.vorba.ch/">Contact</a>.</p>
+        <p>&copy; 2012&ndash;<%= new Date().getFullYear() %> &ndash; Paul Vorbach. <a href="https://paul.vorba.ch/">Contact</a>.</p>
     </footer>
     <script type="text/javascript" src="jquery-2.0.3.min.js"></script>
     <script type="text/javascript" src="highcharts-3.0.10.min.js"></script>
-    <script type="text/javascript" src="moment-2.14.1.min.js"></script>
```
Comment (ogonkov) [inline src/main/html/charts.ejs]: `moment` is bundled in `charts.js` by webpack

**Comment by ogonkov** [comment]
@pvorb can you please look at this PR?

**Comment by pvorb** [comment]
Sorry, I forgot about this PR.

Why did you do all these changes? I don't see the point why the HTML file should be generated. This will only increase the build time, won't it?

**Comment by ogonkov** [comment]
I think it's not increase compilation time a lot (1-2s maybe?).

Having templates generation got it advantages, for example, year update routine could be automated by CI schedule (year is now calculated). Next PRs could split HTML to includes, to avoid code duplication.

I'm also add meta tag to adopt html to mobile.

**Comment by pvorb** [comment]
Ok, the point about the duplicated moment dependency is valid.

**Comment by ogonkov** [comment]
@pvorb that means that other points doesn't work for you?

**Comment by pvorb** [comment]
Well, there's no reason to have the copyright year updated automatically. And since there are only two static HTML files, I don't see why I should add the complexity of a HTML compiler now.

**Comment by ljharb** [comment]
(The copyright "ending" year isn't even necessary; you can just use the starting year and never have to update it again)

**Comment by pvorb** [comment]
I don't even have to add a copyright at all. This tool is APL2-licensed.

**Comment by ogonkov** [comment]
ok, i will close this PR

**Pull request closed without merging**